felles/src/main/kotlin/no/nav/familie/kontrakter/felles/simulering/Simuleringsoppsummering.kt

```diff
@@ -13,6 +13,7 @@ data class Simuleringsoppsummering(
     val forfallsdatoNestePeriode: LocalDate?,
     val tidSimuleringHentet: LocalDate?,
     val tomSisteUtbetaling: LocalDate?,
+    val sumManuellePosteringer: BigDecimal = BigDecimal.ZERO,
```

Comment:
Hvordan vil dette fungere/se ut for gamle (allerede utførte simuleringer)? Behandlinger som har status ferdigstilt?

Reply:
Usikker, må den være nullable eller klarer den å oversette?

Reply:
Setter denne til å være nullable siden den fint kan være det